subscription/src/main/resources/org/killbill/billing/subscription/migration/V20190712115638__subscription_external_key.sql

```diff
@@ -0,0 +1,8 @@
+alter table subscriptions add column external_key varchar(255) after bundle_id;
+update subscriptions set external_key=id;
```

Comment:
To be truly backward compatible, I'm wondering if this shouldn't be set to the bundle external key instead? The query is very hard to write though (with transfers in mind). Something to call out in the release notes.

Reply:
⚠️ Yep, this is currently not backward compatible, and worst than that slightly sneaky because the definition of what `Subscription#externalKey` means has changed. We will for sure need to document it at the very least.